package.json

```diff
@@ -75,18 +75,20 @@
     "ngx-device-detector": "^1.3.0",
     "ngx-md": "~6.0.0",
     "ngx-pagination": "^3.0.3",
+    "node-sass": "^4.9.3",
     "reflect-metadata": "^0.1.12",
     "rx-polling": "~1.0.0",
     "rxjs": "^6.2.1",
     "rxjs-compat": "^6.0.0-rc.0",
+    "sass-loader": "^7.1.0",
```

Comment:
This looks like a development dependency, I don't think it should have been added here?